fenix/app/src/main/java/org/mozilla/fenix/shopping/di/ReviewQualityCheckMiddlewareProvider.kt

```diff
@@ -23,16 +24,21 @@ object ReviewQualityCheckMiddlewareProvider {
      *
      * @param settings The [Settings] instance to use.
      * @param browserStore The [BrowserStore] instance to access state.
+     * @param openLinkInNewTab Opens a link in a new tab.
+     * @param openLinkInSelectedTab Opens a link in the currently selected tab.
      * @param scope The [CoroutineScope] to use for launching coroutines.
      */
     fun provideMiddleware(
         settings: Settings,
         browserStore: BrowserStore,
+        openLinkInNewTab: (String) -> Unit,
+        openLinkInSelectedTab: (String) -> Unit,
```

Comment:
don't have a strong opinion on this, but I could see the argument to have this be a singular lambda with a boolean for whether or not to open this in a new tab.